eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -506,6 +506,12 @@
         "code": "java.annotation.attributeValueChanged",
         "new": "@interface com.azure.core.annotation.JsonFlatten",
         "justification": "Changes made the annotation more permissive on the target types."
+      },
+      {
+        "code": "java.field.serialVersionUIDUnchanged",
+        "old": "field com.azure.core.amqp.exception.AmqpErrorContext.serialVersionUID",
+        "new": "field com.azure.core.amqp.exception.AmqpErrorContext.serialVersionUID",
```

Comment:
This is an easy thing to fix rather than to add a suppression. Just generate a new serial version for the exception.

Reply:
I had changed it before but then got another revapi error that the serialversionUID changed.
1. [serialVersionUIDChanged](https://revapi.org/revapi-java/0.24.2/differences.html#_field_serialversionuid_changed_java_field_serialversionuidchanged)

[serialVersionUIDUNChanged](https://revapi.org/revapi-java/0.24.2/differences.html#_field_serialversionuid_unchanged_java_field_serialversionuidunchanged): This one is "potentially breaking", so I read through the Java serialization guidelines (http://docs.oracle.com/javase/6/docs/platform/serialization/spec/version.html#6678) to see if it was